src/ownership.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import cockpit from 'cockpit';
+
+import { FileInfo } from './fsinfo';
+
+interface Id {
+    id: number;
+    name: string | null;
+}
+
+interface IdInfo {
+    uid: Id;
+    gid: Id;
+    groups: Id[];
+}
+
+function parse_id(content: string): Id {
+    const match = content.match(/^(\d+)\(([^)]+)\)$/);
```

Comment:
I'd suggest:
```
const match = content.match(/^(\d+)\(([^)]+)?\)$/);
```

For the case where the group name is empty.

Reply:
I tested this again locally after updating and now I notice that the behaviour has changed — instead of getting something like `uid=234(unbound)` I get `uid=234` without a section in parens.  We should probably support that by adding a non-capturing group outside of the literal `()` and appending a `?` to it.

Reply:
How do you get that?

Reply:
```
⬢[lis@toolbox ~]$ sudo python3
Python 3.12.3 (main, Apr 17 2024, 00:00:00) [GCC 14.0.1 20240411 (Red Hat 14.0.1-0)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.setuid(234)
>>> os.execlp('id', 'id')
uid=234 gid=0(root) groups=0(root)
```

Reply:
This would be the case where the logged in user doesn't have an entry in `/etc/passwd`.  Pretty marginal case, imho...

Reply:
Hmmmm sounds like this feature needs elaborate test cases :)

Reply:
I'd prefer not.  We have another PR in the bridge which is about to abort the entire bridge in this case.  Let's keep this workaround simple — we'll be removing it soon, anyway.

Reply:
Ok! I can live with that.